test/openAPI/features/support/wallet_generate_link_auth_code.js

```diff
@@ -187,6 +231,150 @@ When(
             })
 );
 
+// Scenario: Not able to validate the link-code and its expiry because of transaction and link code are not connected to each other
+// Given and others Then for this scenario are written in the aforementioned example
+
+When(
+    /^Send POST \/linked\-authorization\/link\-auth\-code request with given valid linkCode and transactionId$/,
+    () =>
+        specWalletGenerateLinkAuthCode.post(baseUrl)
+            .withHeaders(X_XSRF_TOKEN.key, X_XSRF_TOKEN.value)
+            .withJson({
+              requestTime: new Date().toISOString(),
+              request: {
+                linkedCode: 'valid_link_code_001',
```

Comment:
Naming ''valid_link_code_001'' might not ensure that is valid, we have to have a 2nd transaction id, generate separate link code for that, call link-transaction, authenticate and consent for that and then use the 1st trasaction id and 2nd link code to call /link-auth-code. When we follow this, both transaction id and link code are valid with state eligible for /link-auth-code but not connected to each other.
Moving partial comment from https://github.com/GovStackWorkingGroup/bb-identity/pull/34#discussion_r1218345207